tools/tests/_utils/Semver.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*/
+
+import * as semver from "../../src/_utils/semver";
+
+describe('coerce', () => {
+  it ('null', () => {
+    expect(semver.coerce('')).toEqual('')
+    expect(semver.coerce('1.2.3')).toEqual('1.2.3')
+    expect(semver.coerce('1.2')).toEqual('1.2.0')
+    expect(semver.coerce('1')).toEqual('1.0.0')
+  })
+});
+
+describe('satisfies', () => {
+  it ('defaults', () => {
+    expect(semver.satisfies('', '>= 1.3, < 99.0')).toBe(true)
+  })
+
+  it ('semver', () => {
+    expect(semver.satisfies(semver.coerce('2.17.0'), '>= 1.3, < 99.0')).toBe(true)
```

Comment:
Why do we need special comma handling when the correct way to write this according to the [semver package](https://docs.npmjs.com/cli/v6/using-npm/semver#ranges) is to just remove the comma: `>=1.3 <99.0`

Reply:
Hmmmm, interesting. I actually didn’t know that’s correct. I’ll get rid of comma handling and make sure we raise errors if this is incorrectly specified.

Reply:
Updated.